Duplicate linting in latest master

- I'm seeing files being linted twice when building from master branch.

Using this command within a folder: `swiftlint lint`

```
Linting Swift files in current working directory
Linting 'test.swift' (1/2)
/Users/dbeard/Dev/SCSSTest/test.swift:1: warning: Leading Whitespace Violation: File shouldn't start with whitespace: currently starts with 1 whitespace characters
/Users/dbeard/Dev/SCSSTest/test.swift:10: warning: Trailing Newline Violation: File should have a single trailing newline
/Users/dbeard/Dev/SCSSTest/test.swift:4:7: warning: Colon Violation: When specifying a type, always associate the colon with the identifier
Linting 'test.swift' (2/2)
/Users/dbeard/Dev/SCSSTest/test.swift:1: warning: Leading Whitespace Violation: File shouldn't start with whitespace: currently starts with 1 whitespace characters
/Users/dbeard/Dev/SCSSTest/test.swift:10: warning: Trailing Newline Violation: File should have a single trailing newline
/Users/dbeard/Dev/SCSSTest/test.swift:4:7: warning: Colon Violation: When specifying a type, always associate the colon with the identifier
Done linting! Found 6 violations, 0 serious in 2 files.
```

`ls` of the folder:
```
total 40
-rw-r--r--   1 dbeard  GROUPON\Domain Users   258B Sep 22 11:20 .ci.yml
drwxr-xr-x  14 dbeard  GROUPON\Domain Users   476B Oct  6 14:01 .git
-rw-r--r--   1 dbeard  GROUPON\Domain Users    15B Oct  6 13:23 .swiftlint.yml.old
-rw-r--r--   1 dbeard  GROUPON\Domain Users   122B Sep 22 11:08 old.scss
-rw-r--r--   1 dbeard  GROUPON\Domain Users   628B Oct  6 13:25 output.json
-rw-r--r--   1 dbeard  GROUPON\Domain Users    55B Oct  6 13:10 test.swift
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,10 @@
   [JP Simard](https://github.com/jpsim)
   [#151](https://github.com/realm/SwiftLint/issues/151)
 
+* Deduplicate files in the current directory.  
+  [JP Simard](https://github.com/jpsim)
+  [#154](https://github.com/realm/SwiftLint/issues/154)
+
 
 ## 0.2.0: Tumble Dry
 

--- a/Source/SwiftLintFramework/NSFileManager+SwiftLint.swift
+++ b/Source/SwiftLintFramework/NSFileManager+SwiftLint.swift
@@ -10,8 +10,7 @@ import Foundation
 
 extension NSFileManager {
     public func allFilesRecursively(directory directory: String) -> [String] {
-        let relativeFiles = (try! contentsOfDirectoryAtPath(directory)) +
-            (try! subpathsOfDirectoryAtPath(directory))
-        return relativeFiles.map((directory as NSString).stringByAppendingPathComponent)
+        return try! subpathsOfDirectoryAtPath(directory)
+            .map((directory as NSString).stringByAppendingPathComponent)
     }
 }

